README.md

```diff
@@ -1 +1,217 @@
+To use, run the following command from the `run_scripts` directory:
+
+```
+$ ./3_generate_outputs.sh <user> <google-cloud-credentials-file-path> <pipeline-configuration-file-path> <data-root>
+```
+
+where:
+- `user` is the identifier of the person running the script, for use in the TracedData Metadata 
+  e.g. `user@africasvoices.org`.
+- `google-cloud-credentials-file-path` is an absolute path to a json file containing the private key credentials
+  for accessing a cloud storage credentials bucket containing all the other project credentials files.
+- `pipeline-configuration-file-path` is an absolute path to a pipeline configuration json file.
+- `data-root` is an absolute path to the directory in which all pipeline data should be stored.
+  All output files will be saved in `<data-root>/Outputs`.
+   
+As well as uploading the messages, individuals, and production CSVs to Drive (if configured in the 
+pipeline configuration json file), this stage outputs the following files to `<data-root>/Outputs`:
+ - Local copies of the messages, individuals, and production CSVs (`messages.csv`, `individuals.csv`, 
+   `production.csv`)
+ - A serialized export of the list of TracedData objects representing all the data that was exported for analysis 
+   (`messages_traced_data.json` for `messages.csv` and `individuals_traced_data.json` for `individuals.csv`)
+ - For each week of radio shows, a random sample of 200 messages that weren't classified as noise, for use in ICR (`ICR/`)
+ - Coda V2 messages files for each dataset (`Coda Files/<dataset>.json`). To upload these to Coda, see the next step.
+
+### 4. Upload Auto-Coded Data to Coda
+This stage uploads messages to Coda for manual coding and verification.
+Messages which have already been uploaded will not be added again or overwritten.
+To use, run the following command from the `run_scripts` directory:
+
+```
+$ ./4_coda_add.sh <coda-auth-file> <coda-v2-root> <data-root>
+```
+
+where:
+- `coda-auth-file` is an absolute path to the private credentials file for the Coda instance to download coded datasets from.
+- `coda-v2-root` is an absolute path to a local directory containing a clone of the 
+  [CodaV2](https://github.com/AfricasVoices/CodaV2) repository.
+  If the given directory does not exist, the latest version of the Coda V2 repository will be cloned and set up 
+  in that location automatically.
+- `data-root` is an absolute path to the directory in which all pipeline data should be stored.
+  Downloaded Coda files are saved to `<data-root>/Coded Coda Files/<dataset>.json`.
+  
+### 5. Run Automated Analysis
+This stage runs some standard, automated analysis, including generating standard analysis CSVs, charts, and 
+participation maps, and optionally uploads these to Google Drive. To use, run the following command from 
+the `run_scripts` directory:
+
+```
+$ ./5_automated_analysis.sh <user> <google-cloud-credentials-file-path> <pipeline-configuration-file-path> <data-root>
```

Comment:
This doesn't (currently) match what's in the repo. Is the intention to rename this file very soon?

Reply:
@as2388 yes updated here https://github.com/AfricasVoices/Project-COVID19-SOM/pull/6